client/src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import css from "./style.css";
+import styled from "styled-components";
+
+const Title = styled.h1`
+  font-size: 1.5em;
+  color: #443730;
```

Comment:
We sometimes would move colours etc to a shared theme so we can re-use them when needed.

Reply:
@BrownMark thanks for reviewing! I've made these couple of changes 👍